apps/settings/lib/Activity/Provider.php

```diff
@@ -112,9 +112,9 @@ public function parse($language, IEvent $event, IEvent $previousEvent = null): I
 			$subject = $this->l->t('Your email address was changed by an administrator');
 		} elseif ($event->getSubject() === self::APP_TOKEN_CREATED) {
 			if ($event->getAffectedUser() === $event->getAuthor()) {
-				$subject = $this->l->t('You created app password "{token}"');
+				$subject = $this->l->t('You created the session "{token}"');
```

Comment:
why not use the wording you used in the ticket? 
```suggestion
				$subject = $this->l->t('You created a session with "{token}"');
```

Reply:
@miaulalala Because I realized "You created a session with DEVICE_NAME" is not completely accurate as you can manually create app passwords and give them an arbitrary name. App passwords can be created in many other contexts than a client's login flow.
So this message is more generic.
What do you think?

Reply:
hm yes it makes sense from a dev perspective, but it feels like a grammatical error if you don't have the technical background. I leave the decision up to you :)

Reply:
I think it can be useful to have the wording "app password" in the activity to help people relate to the action in settings, but you are right that seeing `You created the app password "Samsung"` is a bit alarming because it makes me think the password here is "Samsung". 

What do you think about: `You created an app password for the session [session name]` ?

Reply:
Or how about we specifically say it's a label? ` You created an app password with session name [session name]`.

Reply:
How about a mix of both your suggestions?
`You created an app password for a session named "{token}"`